strings/tips.txt

```diff
@@ -122,8 +122,11 @@ As the Librarian, be sure to keep the shelves stocked and the library clean for
 As a Cargo Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
 As a Cargo Technician, you can order contraband items from the supply shuttle console by de-constructing it and using a multitool on the circuit board, the re-assembling it.
 As a Cargo Technician, you can earn more cargo points by shipping back crates from maintenance, liquid containers, plasma sheets, rare seeds from hydroponics, and more!
-As a Shaft Miner, the western side of the Asteroid has a lot more rare minerals than on the east, but is a lot more dangerous.
+As a Shaft Miner, northern part of lavaland has a lot more rare minerals than on the southern but is a lot more dangerous.
```

Comment:
As a Shaft Miner, the northern part of Lavaland has a lot more rare minerals than the Southern part, but is a lot more dangerous.